src/toil_scripts/lib/programs.py

```diff
@@ -59,7 +61,7 @@ def docker_call(tool,
         assert(os.path.isfile(os.path.join(work_dir, filename)))
 
     if mock:
-        for filename, url in outputs.items():
+        for filename, url in outputs.iteritems():
```

Comment:
OOC, what's the need for the change here, and have you revalidated that mock mode works?